<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CwmsCreatObjectListDlgClass</class>
 <widget class="QDialog" name="CwmsCreatObjectListDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Objektcontainer anlegen</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>Schl√ºssel</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="m_pqleKeyname"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="textLabel2">
     <property name="text">
      <string>Name</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="m_pqleCaption"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Klasse</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="m_pqleClass">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/emresources.qrc">
       <normaloff>:/icons/search24.png</normaloff>:/icons/search24.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Beschreibung</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QPlainTextEdit" name="m_pqteComment"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <layout class="QHBoxLayout">
     <item>
      <spacer name="spacer4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>111</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pqpbCancel">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/emresources.qrc">
         <normaloff>:/icons/cancel24.png</normaloff>:/icons/cancel24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pqpbOK">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/emresources.qrc">
         <normaloff>:/icons/ok24.png</normaloff>:/icons/ok24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>m_pqleKeyname</tabstop>
  <tabstop>m_pqleCaption</tabstop>
  <tabstop>m_pqteComment</tabstop>
  <tabstop>m_pqpbOK</tabstop>
  <tabstop>m_pqpbCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/emresources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_pqpbCancel</sender>
   <signal>clicked()</signal>
   <receiver>CwmsCreatObjectListDlgClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>CwmsCreatObjectListDlgClass</receiver>
   <slot>SelectClassClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>622</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbOK</sender>
   <signal>clicked()</signal>
   <receiver>CwmsCreatObjectListDlgClass</receiver>
   <slot>OKClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>671</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SelectClassClickedSlot()</slot>
  <slot>OKClickedSlot()</slot>
 </slots>
</ui>
